sdk/dotnet/Provider.cs

```diff
@@ -80,29 +80,11 @@ public InputList<string> ForbiddenAccountIds
             set => _forbiddenAccountIds = value;
         }
 
-        [Input("ignoreTagPrefixes", json: true)]
-        private InputList<string>? _ignoreTagPrefixes;
-
         /// <summary>
-        /// Resource tag key prefixes to ignore across all resources.
+        /// Configuration block with settings to ignore resource tags across all resources.
         /// </summary>
-        public InputList<string> IgnoreTagPrefixes
-        {
-            get => _ignoreTagPrefixes ?? (_ignoreTagPrefixes = new InputList<string>());
-            set => _ignoreTagPrefixes = value;
-        }
-
         [Input("ignoreTags", json: true)]
-        private InputList<string>? _ignoreTags;
-
-        /// <summary>
-        /// Resource tag keys to ignore across all resources.
-        /// </summary>
-        public InputList<string> IgnoreTags
-        {
-            get => _ignoreTags ?? (_ignoreTags = new InputList<string>());
-            set => _ignoreTags = value;
-        }
+        public Input<Inputs.ProviderIgnoreTagsArgs>? IgnoreTags { get; set; }
```

Comment:
Hmmm... this looks like a breaking change?

Reply:
See https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2570-april-09-2020.

This is a breaking change in a "preview feature" - so they believe it is okay for a minor version bump.

Reply:
awesome - merging then